sqlmesh/core/snapshot/definition.py

```diff
@@ -456,21 +454,18 @@ def from_model(
 
         Args:
             model: Model to snapshot.
-            physical_schema: The schema of the snapshot which represents where it is stored.
-            models: Dictionary of all models in the graph to make the fingerprint dependent on parent changes.
-                If no dictionary is passed in the fingerprint will not be dependent on a model's parents.
+            nodes: Dictionary of all nodes in the graph to make the fingerprint dependent on parent changes.
+                If no dictionary is passed in the fingerprint will not be dependent on a node's parents.
             ttl: A TTL to determine how long orphaned (snapshots that are not promoted anywhere) should live.
             version: The version that a snapshot is associated with. Usually set during the planning phase.
             audits: Available audits by name.
             cache: Cache of model name to fingerprints.
-
         Returns:
```

Comment:
Why do you dislike empty lines so much lol?

Reply:
Lol I originally put something there and then removed which consumed the newline 🤷 I added it back in.